src/HW/server/services/itemManager.js

```diff
@@ -150,22 +151,28 @@ class ItemManager {
         where: { pokemonId: pokemonId },
         raw: true,
       });
-      if (itemFromDb!==null&& itemFromDb.pokemonId === pokemonId) return true;
+      if (itemFromDb !== null && itemFromDb.pokemonId === pokemonId) return true;
       else return false;
     } catch (err) {
       throw new Error(err);
     }
   }
-  async  updateStatusInDb(itemId, newStatus){
-  try{
-    let status = newStatus
-    await Item.update({status},{ where: { itemId: itemId } })
-  }
-  catch(err)
-  {
-throw new Error(err)
+  async updateStatusInDb(itemId, newStatus) {
+    try {
+      const status = newStatus;
+      await Item.update({ status }, { where: { itemId: itemId } });
+    } catch (err) {
+      throw new Error(err);
+    }
   }
-  
+
+  async editTaskNameInDb(itemId, newTaskName) {
+    try {
+      const itemName = newTaskName;
+      await Item.update({ itemName }, { where: { itemId: itemId } });
```

Comment:
you can create a generic function that gets params and update the item by id, and reuse it in updateStatusInDb and potentially more places in the future